pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR.pkgproj

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="dir.props" />
+
+  <PropertyGroup>
+    <Version>2.0.0</Version>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <File Include="files\Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR.props">
```

Comment:
The UWPShim and MSbuild tasks/targets are coming form the MSI short term. So, the new testing scenario requires coordination to obtain the new MSI and this new NuGet package. The two .props and .targets file get consumed automatically by NuGet when this meta-package (and transitively the new NuGet package) is referenced.